<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssignDefectDialog</class>
 <widget class="QDialog" name="AssignDefectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assign Defect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="defectListComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="assignToLabel">
     <property name="text">
      <string>Assign To:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="employeeListComboBox"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="submitButton">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
